app/config/collections.php

```diff
@@ -751,6 +751,107 @@
         ],
     ],
 
+    'schedules' => [
+        '$collection' => ID::custom(Database::METADATA),
+        '$id' => ID::custom('schedules'),
+        'name' => 'schedules',
+        'attributes' => [
+            [
+                '$id' => ID::custom('resourceType'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 100,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('resourceId'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('resourceUpdatedAt'),
```

Comment:
Does this column represent the time at which the schedule was updated or the time at which the function / deployment was updated?

Reply:
The time at which the schedule was updated